buildSrc/src/main/kotlin/com/dfsek/terra/DependencyConfig.kt

```diff
@@ -7,7 +7,7 @@ import org.gradle.kotlin.dsl.repositories
 fun Project.configureDependencies() {
 
     repositories {
-        maven { url = uri("https://hub.spigotmc.org/nexus/content/repositories/snapshots/") }
+        maven { url = uri("https://papermc.io/repo/repository/maven-public/") }
```

Comment:
The paper maven repo is already defined later. This is either redundant or a unintended change.